common/ts/publish-task.ts

```diff
@@ -35,7 +35,13 @@ export default async function publishTask(endpointType: EndpointType) {
     globalQualityGateStatus = 'ok';
   }
 
-  fillBuildProperty();
+  tl.debug(
+    `Following QualityGate status : '${globalQualityGateStatus}' has been deducted from ${
```

Comment:
It might be simpler to log two messages:
`Number of analyses in this build: {x}`
`Overall Quality Gate status: {x}`
(or perhaps "Calculated Overall Quality Gate status" or just "Calculated Quality Gate status" instead)
 
FYI it would be "deduced" rather than "deducted", although it would be more usual to say "calculated".

Reply:
Agreed, this has been implemented.